pkg/pilot/genericpilot/genericpilot.go

```diff
@@ -95,8 +103,20 @@ func (g *GenericPilot) runController(stopCh <-chan struct{}) <-chan error {
 	out := make(chan error, 1)
 	go func() {
 		defer close(out)
-		res := g.controller.Run(stopCh)
-		out <- res
+		out <- g.controller.Run(stopCh)
+	}()
+	return out
+}
+
+func (g *GenericPilot) runElector(stopCh <-chan struct{}) <-chan error {
```

Comment:
The stopCh isn't used.
Pass it to `elector.Run` ?

Reply:
`elector.Run` does not accept a stopCh (and is an upstream package) - I included the chan as an arg here for consistency with the other `run*` methods, and in case we do/can use it in future. Can be removed though as like you say, it's not used.